source/_static/custom.css

```diff
@@ -0,0 +1,3 @@
+.wy-nav-content {
+    max-width: 90% !important;
```

Comment:
Curious as to why you needed this.  Something about preventing horizontal scrolling?

Reply:
The output tables are rather wide. I need to remove this and write my own RST table rendering.
`rstcloth.Table` is already broken as it indent is ignored.